### ISSUE TITLE
Error while deploying on ACS

### ISSUE THREAD

**Issue by kunjee17** [SAFE-Stack/SAFE-BookStore#235]
### Description

I was following steps provided on ReadMe for Azure.

### Repro steps

Follow the steps on Read me for Azure other than Storage part.

### Expected behavior

It should run the sample application.

### Actual behavior

Getting following error.
```
502 - Web server received an invalid response while acting as a gateway or proxy server.
There is a problem with the page you are looking for, and it cannot be displayed. When the Web server (while acting as a gateway or proxy) contacted the upstream content server, it received an invalid response from the content server.
```

### Known workarounds

None. Try to redeploy docker image to my account but doesn't work

**Comment by kunjee17** [comment]
I need to add `Server.dll` in start up file. I found out from image. (Should be added to readme?) 

Then getting new error `Service Unavailable` . I tried running docker locally but server is running on 8085 instead of 8080. Am I missing something?

**Comment by forki** [comment]
Yes please add everything to the readme that was not clear.

Am 04.11.2017 05:09 schrieb "Kunjan" <notifications@github.com>:

I need to add Server.dll in start up file. I found out from image. (Should
be added to readme?)

Then getting new error Service Unavailable . I tried running docker locally
but server is running on 8085 instead of 8080. Am I missing something?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/SAFE-Stack/SAFE-BookStore/issues/235#issuecomment-341870306>,
or mute the thread

**Comment by kunjee17** [comment]
@forki will update read me. What about port number? I guess because of suave running 8085, I was unable to access site even after giving Server.dll to start. What I should be doing there?

**Comment by jeremyabbott** [comment]
I'm having the same problem... I found this: https://docs.microsoft.com/en-us/azure/app-service/containers/tutorial-custom-docker-image. I followed the instructions to set up the port, but it still doesn't work. I'm not sure what else needs to be done. You would also then need to map 443 to another port and expose another port in the docker file if you turned SSL on, but it's not clear where you would do that.

**Comment by jeremyabbott** [comment]
tl;dr version:
`az webapp config appsettings set --resource-group <resource group name> --name <app service name> --settings WEBSITES_PORT=8085` needs to be run in the Azure CLI (easiest way is via Azure Cloud Shell from Azure Portal).

**Comment by kunjee17** [comment]
I guess @jeremyabbott 's [post](http://jeremyabbott.github.io/blog/2017/12-09-implementing/index.html) is solving this issue. We can put this in read me and close it. If that is OK.

**Comment by forki** [comment]
Sure. Please send a pull request

Am 20.12.2017 05:58 schrieb "Kunjan" <notifications@github.com>:

I guess @jeremyabbott <https://github.com/jeremyabbott> 's post
<http://jeremyabbott.github.io/blog/2017/12-09-implementing/index.html> is
solving this issue. We can put this in read me and close it. If that is OK.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/SAFE-Stack/SAFE-BookStore/issues/235#issuecomment-352965016>,
or mute the thread

**Comment by jeremyabbott** [comment]
@kunjee17 @forki I [updated the readme](https://github.com/SAFE-Stack/SAFE-BookStore/pull/251) (that's been merged in) with a blurb about updating the HTTP binding. If you want to reference that too that's cool though.
